package.json

```diff
@@ -83,19 +84,18 @@
     "mc_dir": "../mozilla-central"
   },
   "scripts": {
-    "mochitest": "(cd $npm_package_config_mc_dir && ./mach mochitest browser/extensions/activity-stream/test/functional/mochitest )",
-    "mochitest-debug": "(cd $npm_package_config_mc_dir && ./mach mochitest --jsdebugger browser/extensions/activity-stream/test/functional/mochitest )",
-    "cleanmc": "rimraf $npm_package_config_mc_dir/browser/extensions/activity-stream",
+    "mochitest": "(cd $npm_package_config_mc_dir && ./mach mochitest browser/extensions/activity-stream/system-addon/test/functional/mochitest )",
+    "mochitest-debug": "(cd $npm_package_config_mc_dir && ./mach mochitest --jsdebugger browser/extensions/activity-stream/system-addon/test/functional/mochitest )",
+    "cleanmc": "rimraf $npm_package_config_mc_dir/browser/extensions/activity-stream/system-addon",
     "buildmc": "npm-run-all buildmc:*",
     "prebuildmc": "npm run cleanmc",
     "buildmc:locales": "pontoon-to-json --src locales --dest system-addon/data",
     "buildmc:webpack": "webpack --config webpack.system-addon.config.js --display-optimization-bailout",
     "buildmc:css": "node-sass --source-map true --source-map-contents system-addon/content-src/styles -o system-addon/css",
     "buildmc:html": "rimraf system-addon/prerendered && webpack --config webpack.prerender.config.js && node ./bin/render-activity-stream-html.js",
-    "buildmc:copy": "cpx \"system-addon/**/{,.}*\" $npm_package_config_mc_dir/browser/extensions/activity-stream",
-    "buildmc:version": "node ./bin/update-version.js $npm_package_config_mc_dir/browser/extensions/activity-stream",
+    "buildmc:copy": "node ./bin/mirror-trees.js",
```

Comment:
`cpx` is just using glob, which you're separately depending on now. You should be able to exclude directories with:
`cpx './{{,.}*,!(logs|node_modules)/**}' $npm_package_config_mc_dir/…`